Documentation/ReadingContentRecords/Index.rst

```diff
@@ -30,31 +30,33 @@ After creating the :ref:`PAGE <t3tsref:page>` object, we use the :ref:`CONTENT
 content element we use the :ref:`TEXT <t3tsref:cobj-text>` object to perform
 the actual rendering::
 
-    page = PAGE
-    page.typeNum = 0
+.. code-block:: typoscript
```

Comment:
This broke the rendering. As above are two colons `::` this already starts syntax highlighting.
The default syntax is set to TypoScript. So adding this will break the rendering as indentation of this line is wrong. If this line is indented, it will be rendered as TypoScript.

Still I will merge the request and fix this afterwards.